include/unifex/task.hpp

```diff
@@ -152,9 +161,13 @@ struct _task<T>::type {
       return p.stoken_;
     }
 
-    enum class state { empty, value, exception };
+    enum class state { empty, value, exception, done };
+
+    using done_callback = coro::coroutine_handle<>(void*) noexcept;
 
     coro::coroutine_handle<> continuation_;
+    done_callback* doneCallback_ = nullptr;
```

Comment:
Did we ever consciously choose `camelCase` for variables? My preference is for `snake_case`. Obviously not a problem you should be solving here, though.

Reply:
I have always been writing `camelCase` for non-public variable names.
Public data-members that form part of the interface are still `snake_case`.
Although I've noticed others have been using `snake_case` variable names, so we're not consistent.